src/codemagic/cli/cli_help_formatter.py

```diff
@@ -14,14 +18,28 @@ def __init__(self, *args, **kwargs):
         else:
             self._width = sys.maxsize
 
+    @classmethod
+    def suppress_action(cls, action_name: str):
+        cls._suppressed_actions.add(action_name)
+
+    def _exclude_suppressed_actions(self, message: str) -> str:
```

Comment:
I was wondering if there is a way to avoid adding deprecated actions, so that we don't have to suppress them later, if not, let's add a comment justifying the usage of this function.

Reply:
We still need to add those actions so that `argparse` would be aware of them. Otherwise we'd fail prematurely with
```shell
<tool-name> error: argument action: invalid choice: 'deprecated-action-name' (choose from ...)
```